CONTRIBUTING.md

```diff
@@ -10,12 +10,12 @@ Note that contributions may be rejected on the basis of a contributor failing to
 2. If you have direct access to the repository, **create a branch for your changes** and create a pull request for that branch. If not, create a branch on a fork of the repository and create a pull request from there.
     * It's common practice for a repository to reject direct pushes to `master`, so make branching a habit!
     * If PRing from your own fork, **ensure that "Allow edits from maintainers" is checked**. This gives permission for maintainers to commit changes directly to your fork, speeding up the review process.
-3. **Adhere to the prevailing code style**, which we enforce using [flake8](http://flake8.pycqa.org/en/latest/index.html).
-    * Run `flake8` against your code **before** you push it. Your commit will be rejected by the build server if it fails to lint.
-    * [Git Hooks](https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks) are a powerful tool that can be a daunting to set up. Fortunately, [`pre-commit`](https://github.com/pre-commit/pre-commit) abstracts this process away from you and is provided as a dev dependency for this project. Run `pipenv run precommit` when setting up the project and you'll never have to worry about breaking the build for linting errors.
+3. **Adhere to the prevailing code style**, which we enforce using [`flake8`](http://flake8.pycqa.org/en/latest/index.html) and [`pre-commit`](https://pre-commit.com/).
+    * Run `flake8` and `pre-commit` against your code **before** you push it. Your commit will be rejected by the build server if it fails to lint.
+    * [Git Hooks](https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks) are a powerful tool that can be a daunting to set up. Fortunately, `pre-commit` abstracts this process away from you and is provided as a dev dependency for this project. Run `pipenv run precommit` when setting up the project and you'll never have to worry about breaking the build for linting errors.
```

Comment:
```suggestion
    * [Git Hooks](https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks) are a powerful tool that can seem daunting to set up. Fortunately, `pre-commit` abstracts this process away from you and is provided as a dev dependency for this project. Run `pipenv run precommit` when setting up the project and you'll never have to worry about breaking the build for linting errors.
```

The `can be a daunting to set up` is totally wrong and at the least should be fixed, but here are some thoughts on this line:

1. The hooks themselves are really a git feature, and what the hooks execute are the tools. Maybe it should say `features` instead of `tools`.
2. Collectively, hooks are a single feature. Yet, writing plural `features` feels less awkward to read: `hooks are a feature` vs `hooks is a feature` vs `hooks are features`. If we consider them tools rather than features, then it makes more sense to use the plural `tools` rather than the singular.

Reply:
Changes made to `master` after internal review!